Sources/NIO/NIOAny.swift

```diff
@@ -77,6 +77,7 @@ public struct NIOAny {
     /// Try unwrapping the wrapped message as `ByteBuffer`.
     ///
     /// returns: The wrapped `ByteBuffer` or `nil` if the wrapped message is not a `ByteBuffer`.
+    @_versioned
```

Comment:
@Lukasa why not `@_inlineable` as well?

Reply:
Not allowed on public methods according to my compiler.

Reply:
The current reason is that `storage` is `private`. Do we think that's worth changing?

Reply:
@Lukasa not for this particular one but possibly for the ones that take generics.

Reply:
Yeah, I'm a bit on the fence here. I suppose making the storage internal won't be too big a deal.

Reply:
me too tbh. I'll make it your call. If you make the storage internal, please underscore it

Reply:
Ok, made all of these `@_inlineable`.